Makefile

```diff
@@ -22,4 +22,10 @@ test-build:
 	docker compose -f postgres.yml down
 	docker compose -f postgres.yml up --build
 
+python-docker-build:
+	docker-compose build python-app
```

Comment:
iirc it should be `docker compose`

Reply:
yeah sorry, I'm used to the older version of docker compose, we should be using the newer `docker compose` instead where we can